### ISSUE TITLE
no composer support?

### ISSUE THREAD

**Issue by rahilwazir** [immobiliare/ApnsPHP#102]
`composer require duccio/apns-php` doesn't work?

**Comment by smartssa** [comment]
It doesn't appear to. But specifying "dev-master" as a version in composer.json does work.

**Comment by iquito** [comment]
Not having a version number for this library is quite impractical - maybe it would be possible to version the releases? Using dev-master as a version is not good practice and could potentially break when this library is updated/changed.

See also http://www.lornajane.net/posts/2015/relying-on-a-dev-master-dependency-in-composer for some more arguments against dev-master dependencies.

**Comment by wassafr** [comment]
I agree. I created a library that has pans-php has a dependency and I'm always forced to add a specific dependency for pans-php at the root package.

**Comment by iquito** [comment]
With the latest version a version number (v1.0.1) was added for packagist & composer. Thanks!

**Comment by olyckne** [comment]
And since v1.0.1 there hasn't been any more releases which means I can't use some of the new features.
Are we going to get a new release or do I need to find another package for this?
